### ISSUE TITLE
moses.h L02E12

### ISSUE THREAD

**Issue by booleand** [mikeudacity/robo-pace-betatest#114]
- Although Test run outputs images, it also returns the following error:

"libdc1394 error: Failed to initialize libdc1394"

- Could it be helpful to have titles for the images so those unfamiliar with CV can understand what they're looking at and for?

**Comment by cangelogamboa** [comment]
Good idea, I will look into it. Thanks for bringing it to my attention.
